requirements.txt

```diff
@@ -113,6 +126,8 @@ traitlets==5.11.2
     # via
     #   ipython
     #   matplotlib-inline
+urllib3==2.2.1
+    # via requests
```

Comment:
I generated this file using `pip-compile requirements.in ` though there are several more changes than I was expecting 🤔